shared/settings/advanced/index.js

```diff
@@ -17,37 +17,50 @@ type Props = {
   onTrace: (durationSeconds: number) => void,
   onProcessorProfile: (durationSeconds: number) => void,
   onBack: () => void,
+  setLockdownModeError: string,
+  settingLockdownMode: boolean,
   traceInProgress: boolean,
   processorProfileInProgress: boolean,
   hasRandomPW: boolean,
 }
 
-const Advanced = (props: Props) => (
-  <Kb.Box style={styles.advancedContainer}>
-    <Kb.Box style={styles.checkboxContainer}>
-      <Kb.Checkbox
-        checked={!!props.lockdownModeEnabled}
-        disabled={props.lockdownModeEnabled == null || props.hasRandomPW}
-        label={
-          'Forbid account changes from the website' +
-          (props.hasRandomPW ? ' (you need to set a passphrase first)' : '')
-        }
-        onCheck={props.onChangeLockdownMode}
-        style={styles.checkbox}
-      />
-    </Kb.Box>
-    {!Styles.isMobile && !isLinux && (
-      <Kb.Box style={styles.openAtLoginCheckboxContainer}>
+const Advanced = (props: Props) => {
+  const disabled = props.lockdownModeEnabled == null || props.hasRandomPW || props.settingLockdownMode
+  return (
+    <Kb.Box style={styles.advancedContainer}>
+      <Kb.Box style={styles.checkboxContainer}>
         <Kb.Checkbox
-          label="Open Keybase on startup"
-          checked={props.openAtLogin}
-          onCheck={props.onSetOpenAtLogin}
+          checked={!!props.lockdownModeEnabled}
+          disabled={disabled}
+          label={
+            'Forbid account changes from the website' +
+            (props.hasRandomPW ? ' (you need to set a passphrase first)' : '')
+          }
+          onCheck={props.onChangeLockdownMode}
+          style={styles.checkbox}
         />
       </Kb.Box>
-    )}
-    <Developer {...props} />
-  </Kb.Box>
-)
+      <Kb.Box style={styles.progressErrorContainer}>
+        {props.settingLockdownMode && <Kb.ProgressIndicator />}
```

Comment:
This looks out of place before you know the error will go there, maybe it should be above the checkbox
<img width="448" alt="image" src="https://user-images.githubusercontent.com/11968340/52129640-04945500-2606-11e9-84e6-2ad8c3e71bbb.png">

Reply:
Done!